src/transformers/tokenization_bert.py

```diff
@@ -338,7 +349,8 @@ def tokenize(self, text, never_split=None):
         for token in orig_tokens:
             if self.do_lower_case and token not in never_split:
                 token = token.lower()
-                token = self._run_strip_accents(token)
+                if not self.keep_accents:
```

Comment:
If you put this line inside the `if self.do_lower_case and token not in never_split:`, the action of strip accents won't work if I do: `keep_accents=False and do_lower_case=False`. I suggest indent this line at the same level of line 350, so you can only apply lower case, only strip accents or both.